protocol/0057-TRAN-transfers.md

```diff
@@ -156,6 +156,7 @@ Note: if there is no market with contribution to the reward metric - no transfer
 A fee is taken from all transfers, and paid out to validators in a similar manner to the existing [infrastructure fees](0061-REWP-pos_rewards.md). For recurring transfers, the fee is charged each time the transfer occurs.
 
 The fee is set by the `transfer.fee.factor` [network parameter](#network-parameters) that defines the proportion of each transfer taken as a fee.
+The fee is capped by the `transfer.fee.Maxfactor` network parameter to prevent people from paying a large transfer fee amount.
```

Comment:
Should probably be something like `transfer.fee.maxQuantumAmount` as I expect we want it determined by some multiplier of a quantum?